SendUserDataProcessingEmailActivity/index.ts

```diff
@@ -28,6 +34,23 @@ const documentClient = new DocumentDBClient(cosmosDbUri, {
 
 const profileModel = new ProfileModel(documentClient, profilesCollectionUrl);
 
+const findOneProfileByFiscalCodeTask = (pm: ProfileModel) => (
+  fiscalCode: FiscalCode
+) =>
+  TE.tryCatch(
+    () => pm.findOneProfileByFiscalCode(fiscalCode),
+    toError
+  ).foldTaskEither<Error, Option<RetrievedProfile>>(
+    err => TE.left(new Task(async () => err)),
+    queryErrorOrMaybeProfile =>
```

Comment:
all this mess is just to convert `QueryError` to a simple `Error` since we don't need to discriminate + the promise to a taskeither